pkg/app/accounts/usernameonly.go

```diff
@@ -44,9 +50,65 @@ func (u *UsernameOnlyUser) Username() string { return u.username }
 func (u *UsernameOnlyUser) Email() string { return "" }
 
 func userFromRequest(r *http.Request) (*UsernameOnlyUser, error) {
+	// Accept putting the username in a cookie to support using a browser
+	// to interact with CO.
+	cookie, err := r.Cookie(unameCookie)
+	if err == nil && cookie.Value != "" {
+		return &UsernameOnlyUser{cookie.Value}, nil
+	}
 	username, _, ok := r.BasicAuth()
 	if !ok {
 		return nil, apperr.NewBadRequestError("No username in request", nil)
 	}
 	return &UsernameOnlyUser{username}, nil
 }
+
+type LoggingData struct {
+	Username string
+	Error    string
+}
+
+var loggingTemplate = template.Must(template.New("logging").Parse(`
+<!DOCTYPE html>
+<html>
+<head><title>AM Logging</title></head>
+<body>
+    {{if .Error}}
+        <p style="color: red; font-size:2vw;">{{.Error}}</p>
+    {{end}}
+    {{if .Username}}
+        <h2> UsernameOnly account manager</h2>
+        <p style="font-size:3vw;">Welcome {{.Username}}!</p>
+        <p style="font-size:2vw;">You can now visit other pages.</p>
+    {{else}}
+        <form method="POST">
+            <h2> Logging username for UsernameOnly account manager</h2>
+            <label for="uname">username:</label>
+            <input type="text" id="uname" name="username"><br><br>
```

Comment:
nits: add `required`

Reply:
in `<input type="text" id="uname" name="username">`

Reply:
Done.